lib/CMakeLists.txt

```diff
@@ -569,10 +569,11 @@ install(TARGETS volk
 #probably doesn't.
 if(ENABLE_STATIC_LIBS)
   add_library(volk_static STATIC $<TARGET_OBJECTS:volk_obj>)
-  target_link_libraries(volk_static PUBLIC ${volk_libraries} pthread)
+  target_link_libraries(volk_static PUBLIC ${volk_libraries} PRIVATE cpu_features)
```

Comment:
With #420 merged, the `cpu_features` needs to be added within an `if(VOLK_CPU_FEATURES)` clause.

Reply:
Updated.